### ISSUE TITLE
bad result for fine-tuning nasnet-large on customized dataset

### ISSUE THREAD

**Issue by neuhzhj2012** [tensorflow/models#3052]
Nasnet-A_large has better result on Imagenet than Inception*. But After fine-tuning the Nasnet-A_large on my own dataset for 90,000 steps,  the result of evaluation is top1=0.026 and top5=0.098.  The same issue is posted on [stackoverflow](https://stackoverflow.com/questions/47965511/low-accuracy-after-fine-tuning-nasnet-large-compared-to-inception-v4-on-customiz).

**Comment by skye** [comment]
Closing since this is not a bug or feature request.
